pkgs/development/libraries/vulkan-loader/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchgit, fetchFromGitHub, cmake, pkgconfig, git, python3,
+  python3Packages, glslang, spirv-tools, x11, libxcb, wayland }:
+
+assert stdenv.system == "x86_64-linux";
+
+let
+  version = "1.0.26.0";
+  src = fetchFromGitHub {
+    owner = "KhronosGroup";
+    repo = "Vulkan-LoaderAndValidationLayers";
+    rev = "sdk-${version}";
+    sha256 = "157m746hc76xrxd3qq0f44f5dy7pjbz8cx74ykqrlbc7rmpjpk58";
+  };
+  getRev = name: builtins.substring 0 40 (builtins.readFile "${src}/${name}_revision");
```

Comment:
This is no good, as it makes evaluation-time stuff (the assertions) depend on build-time stuff (the `src` download). We want to avoid that in nixpkgs, as the consequence is that nix would start building in unexpected situations, such as just making queries about packages. That's also why it fails on CI and it would fail on Hydra.